bin/cli.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const readline = require('readline');
+
+const Cli = require('../lib/cli/cli');
+
+let rl = null;
+
+const log = (msg) => {
+  const userInput = rl.line;
+  if (userInput) rl.clearLine();
+  console.log(msg);
+  rl.write('\n');
+  if (userInput) rl.write(userInput);
+};
+
+const finish = () => {
+  rl.close();
+  process.exit();
+};
+
+const cli = new Cli(log);
```

Comment:
@aqrln I'm not sure if it'll be better for abstraction as now Cli only depends on the function to output data and not on the  whole readline interface. Also there is another problem - readline needs `completer` upon its creation which is only present after Cli has been created.